src/Runtime/Runtime/Core/Main/INTERNAL_Simulator.cs

```diff
@@ -109,22 +109,25 @@ public static bool IsRunningInTheSimulator_WorkAround
             set;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public static Action<Action> OpenSilverDispatcherBeginInvoke
         {
             set;
             internal get;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public static Action<Action, TimeSpan> OpenSilverDispatcherInvoke
         {
             set;
             internal get;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public static Func<bool> OpenSilverDispatcherCheckAccess
         {
-            get;
-            internal set;
+            set;
+            internal get;
```

Comment:
Let's keep the getter public to avoid unnecessary breaking change
```suggestion
            get;
```